javascript/migratory_birds.js

```diff
@@ -0,0 +1,27 @@
+// Problem: https://www.hackerrank.com/challenges/migratory-birds/
+
+function migratoryBirds(arr) {
+    let max = 0
+    let birds = {}
+    let common = null
+
+    for (let a = 0; a <= arr.length; a++) {
```

Comment:
Este *for loop* está iterando hasta fuera de los límites del arreglo, debería ser:
```suggestion
    for (let a = 0; a < arr.length; a++) {
```

Reply:
Hecho d231af9